install.sh

```diff
@@ -3,7 +3,7 @@
 { # this ensures the entire script is downloaded #
 
 nvm_has() {
-  type "$1" > /dev/null 2>&1
+  command which "${1-}" > /dev/null 2>&1
```

Comment:
this is very intentional, because of how type vs which interacts with functions and aliases.